collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -767,6 +767,8 @@ techo() {
 
 }
 
+LANG=C
```

Comment:
Could we perhaps move the variable to the top of the script, with a comment to describe the use case? It's just a bit hidden and could be called out in case it causes some adverse effect in other environments.

Reply:
Thank you for comment! I fixed it.